.circleci/config.yml

```diff
@@ -92,12 +92,8 @@ jobs:
       - store_artifacts:
           path: /tmp
 
-  e2e-pilot-noauth-v1alpha3-v2:
+  e2e-pilot-auth-v1alpha3-v2:
```

Comment:
why are you editing this?

Reply:
This is just a bad diff by github. In TOC it was decided to run the simple tests and pilot auth test in circle postsubmit.

The diff is weird because I added the auth test and removed the noauth test so github matched them up. The test itself is copied from the old circle config that was previously deleted, nothing really changed